app/src/main/java/com/microsoft/office365/msgraphsnippetapp/SignInActivity.java

```diff
@@ -89,7 +99,7 @@ public void onSuccess(AuthenticationResult authenticationResult) {
     }
 
     @Override
-    public void onError(Exception e) {
+    public void onError(MsalException e) {
```

Comment:
This is a nitpick.

Give variables descriptive names. `error` is more descriptive when compared with `e`

Reply:
Resolved with `exception` to standardize with code base in other classes